README.md

```diff
@@ -50,7 +50,7 @@ jobs:
 
     - name: Setup LXD
       uses: canonical/setup-lxd@[version]
-      # uses pre-installed LXD snap (v4.0.9)
+      # uses pre-installed LXD snap (5.0/stable)
```

Comment:
Pretty sure 20.04 ships with LXD 4.0.9, maybe change the runner version to 20.04 here instead?

Reply:
Think it's more clear to have a contrast with the other example

Reply:
> Pretty sure 20.04 ships with LXD 4.0.9, maybe change the runner version to 20.04 here instead?

Indeed, 20.04 ships with `4.0/stable` which currently has 4.0.9 but the job has `runs-on: ubuntu-22.04` which means it comes `5.0/stable` pre-installed.

Reply:
> Think it's more clear to have a contrast with the other example

Sure, having both example end up using `5.0/stable` is probably not ideal. How about using `latest/stable` for the 22.04 one?

Reply:
Sounds good.

Reply:
I settled on a slightly different version, please let me know what you think.